docs/source/guides/installation_guide.md

```diff
@@ -1,41 +1,76 @@
 # Installation Guide
 
-## Pip Installation
 
-`pip install nucypher`
+## Standard Installation
 
-## Pipenv Installation
+We recommend installing nucypher with either `pip` or `pipenv`
```

Comment:
Are we killing pipenv? IIRC @tuxxy and @jMyles discussed removing pipenv in discord but not sure if that has happened.

Reply:
As I stand now - I use both workflows effectively.